validator/index.ts

```diff
@@ -0,0 +1,48 @@
+import blgr from 'blgr';
+import ethers from "ethers";
+import to from 'await-to-js';
+import {ValidatorCtx} from "./types";
+import {unlockWallet} from "./utils/wallet";
+const logger = blgr.logger('debug');
```

Comment:
I think we should pass a logger instance into the object below, and then use that instance everywhere.
That way we can configure it once and have the setting used everywhere.
maybe something like
```javascript
constructor(..., logger) {
  this.logger = logger;
}

/// later
this.logger.info("Unlocking wallets...");
```